docs/pages/index.tsx

```diff
@@ -32,82 +33,59 @@ function loadDependencies() {
   loadScript('https://platform.twitter.com/widgets.js', document.querySelector('head'));
 }
 
-const useStyles = makeStyles(
-  (theme) => ({
-    root: {
-      flex: '1 0 100%',
-    },
-    hero: {
-      paddingTop: theme.spacing(8),
-      color: theme.palette.primary.main,
-    },
-    content: {
-      display: 'flex',
-      flexDirection: 'column',
-      alignItems: 'center',
-      textAlign: 'center',
-      paddingTop: theme.spacing(4),
-      paddingBottom: theme.spacing(8),
-      [theme.breakpoints.up('md')]: {
-        paddingTop: theme.spacing(16),
-        paddingBottom: theme.spacing(16),
-        flexDirection: 'row',
-        alignItems: 'flex-start',
-        textAlign: 'left',
-      },
-    },
-    title: {
-      marginLeft: -12,
-      whiteSpace: 'nowrap',
-      letterSpacing: '.7rem',
-      textIndent: '.7rem',
-      fontWeight: theme.typography.fontWeightLight,
-      [theme.breakpoints.only('xs')]: {
-        fontSize: 28,
-      },
-    },
-    logo: {
-      flexShrink: 0,
-      width: 120,
-      height: 120,
-      marginBottom: theme.spacing(2),
-      [theme.breakpoints.up('md')]: {
-        marginRight: theme.spacing(8),
-        width: 195,
-        height: 175,
-      },
-    },
-    button: {
-      marginTop: theme.spacing(4),
-    },
-    social: {
-      padding: theme.spacing(2, 0),
-      display: 'flex',
-      justifyContent: 'center',
-      alignItems: 'center',
-      minHeight: 21,
-      boxSizing: 'content-box',
-      '& a': {
-        color: theme.palette.background.paper,
-      },
-    },
-    github: {
-      width: 105,
-      display: 'flex',
-      justifyContent: 'flex-end',
-      marginRight: theme.spacing(1),
-      '& span': {
-        display: 'flex',
-      },
-    },
-    twitter: {
-      width: 160,
-      display: 'flex',
+const Content = styled(Container)(({ theme }) => ({
```

Comment:
This page is going away, it would have been simpler to change the import only (it seems).

Reply:
Oh well..